VoiceOverlay/Controllers/VoiceOverlayController.swift

```diff
@@ -27,7 +29,15 @@ import Speech
     public var datasource: Any? = nil
   
     public override init() {
-        super.init()
+      self.recordableHandler = {
+        return SpeechController()
+      }
+      super.init()
+    }
+  
+    public init(speechControllerHandler: @escaping RecordableHandler) {
```

Comment:
For a better DX, it might be better to have `Recordable` as a parameter, and then construct the handler inside the `init`

Reply:
What do you mean ?

Reply:
Oh, I understand. But since the usage today is: every time you create a STT overlay, we re-init the class, passing the parameter will block us from doing so

Reply:
ahh ok got it, then keep as is 👍